tools/Raven.Debug/LogTrafficWatch/TrafficWatchReplay.cs

```diff
@@ -278,6 +286,33 @@ private static string ConvertJsonElement(JsonElement e)
                     _ => $"\"{e}\""
                 };
             }
+
+            private static void OutputLogToConsole(HttpResponseMessage response, byte[] responseBuffer, TrafficWatchHttpChange item)
+            {
+                StringBuilder sb = new ();
+                
+                sb.AppendLine("Failed to execute request");
+                
+                sb.AppendLine($"Http method: {item.HttpMethod}");
+                
+                sb.AppendLine($"Request Uri: {item.RequestUri}");
+                
+                sb.AppendLine($"Absolute Uri: {item.AbsoluteUri}");
+                        
+                sb.AppendLine($"Custom info: {item.CustomInfo}");
+                        
+                sb.AppendLine($"Status code: {(int)response.StatusCode}");
+                
+                sb.AppendLine($"Request size in bytes: {item.RequestSizeInBytes}");
+                
+                sb.AppendLine($"Response size in bytes: {item.ResponseSizeInBytes}");
+                
+                sb.AppendLine($"Request Id: {item.RequestId}");
+                    
+                sb.AppendLine(Encoding.Default.GetString(responseBuffer));
```

Comment:
It seems that the consumption of the stream is done without really saving the result.
It overwrites the preview bytes (no use of offset) each time.
If you want to read it here you need to properly consume the result (and also use the length of the read bytes here)